recipes/sumaclust/conda_build.sh

```diff
@@ -0,0 +1,4 @@
+
+bash conda_build.sh
```

Comment:
conda_build.sh is not part of this PR.
I have checked the tarball and there is a makefile included. Does `make install` work instead?

Reply:
`make` by itself is what is used to install it locally.

Reply:
Thanks for your help @bgruening!  I tried naming the file `build.sh` and it fails (wish some sort of hint to rename it as `conda_build.sh`, and when I do name it `conda_build.sh` it works for me locally. Everything is green before I change things. I attempted to add `make` into the bash script because that is how to install it locally. BTW I'm definitely more bioinformatics than CS if I sound confused. :grin: I have a few more tools I need to get into Bioconda, so any help is great so I can repeat the process. 

Thank you!